config

```diff
@@ -125,6 +125,9 @@
     # Fetch PR from GitHub by number/id
     fetchpr = "!sh -c 'git fetch origin pull/$0/head:pr/$0'"
 
+    # View different commits between branches
+    diffbranch = log --pretty=format:'%C(yellow)%h %Cred%ad %Cblue%an%Cgreen%d %Creset%s' --abbrev-commit --date=iso --graph
```

Comment:
No, you're correct - `hist` will do the same if branch names provided. I've overlooked it. Sorry for bothering & to be closed ;)